src/Microsoft.Data.Analysis/DataFrameColumn.cs

```diff
@@ -24,6 +24,49 @@ public DataFrameColumn(string name, long length, Type type)
             DataType = type;
         }
 
+        #region Static Factory Methods
```

Comment:
Nit remove the `#region`, it not necessary and not used anywhere else.

Reply:
Changed.